<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>860</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: &quot;Source Code Pro&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_sifrovani">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Šifrování</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zadej_otevreny">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Zadej otevřený text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filtrovany">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Filtrovaný text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sifrovany">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Šifrovaný text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_desifrovani">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Dešifrování</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zadej_sifrovany">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>60</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Zadej šifrovaný text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_desifrovany">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>220</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Dešifrovaný text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_otevreny">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>380</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Otevřený text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_klic">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>KLÍČ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_otevreny">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_key">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_sifrovany">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>80</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_filtrovany">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_sifrovany">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_desifrovany">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>240</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_otevreny">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>400</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_zasifruj">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>310</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Zašifruj text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_desifruj">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Dešifruj text</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>641</width>
     <height>211</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>621</width>
     <height>211</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(200, 200, 200);
border: none;
max-length: 1;
</string>
   </property>
   <row>
    <property name="text">
     <string>A</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>D</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>F</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>G</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>V/X</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>X</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>A</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>D</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>F</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>G</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>V/X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
  </widget>
  <widget class="QRadioButton" name="radio_cz">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ADFGX (CZ)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_en">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ADFGX (EN)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_jazyk">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ŠIFRA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_klic_upozorneni">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&gt; min. 8 znaků (bez duplicity a pouze písmena A-Z!)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_over_klic">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>150</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt; Ověřit klíč &gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_ano">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Source Code Pro&quot;;
color: rgb(85, 255, 0);
text-align: center;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_ne">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Source Code Pro&quot;;
color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tabulka">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>TABULKA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_tabulka_upozorneni">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&gt; do každého pole právě 1 znak, bez duplicity - každý znak musí být jedinečný</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_tabulka_upozorneni_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&gt; ADFGX: zadáváte VELKÁ písmena A-Z (pozor na jazyk! CZ: bez W; EN: bez J)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_tabulka_upozorneni_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&gt; ADFGVX: zadáváte VELKÁ písmena A-Z + číslice 0-9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_over_tabulku">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>480</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt; Ověřit tabulku &gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_ano_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>480</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 255, 0);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output_ne_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>480</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_upozorneni">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&gt; před (de)šifrováním se ujistěte, zda je správně vyplněný klíč a tabulka</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_v">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ADFGVX</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_generovat">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>590</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;&lt; Generovat tabulku</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_vymazat">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>650</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;&lt; Vymazat tabulku</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_upozorneni_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>&gt; písmena české abecedy jsou převedena na anglický ekvivalent!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
